webapp/pages/settings/index.vue

```diff
@@ -60,9 +71,10 @@ const query = gql`
 */
 
 const mutation = gql`
-  mutation($id: ID!, $name: String, $locationName: String, $about: String) {
-    UpdateUser(id: $id, name: $name, locationName: $locationName, about: $about) {
+  mutation($id: ID!, $slug: String, $name: String, $locationName: String, $about: String) {
+    UpdateUser(id: $id, slug: $slug, name: $name, locationName: $locationName, about: $about) {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-23T11:59:24Z" title="Monday, September 23rd 2019, 1:59:24 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -60,9 +71,10 @@ const query = gql`
 */
 
 const mutation = gql`
-  mutation($id: ID!, $name: String, $locationName: String, $about: String) {
-    UpdateUser(id: $id, name: $name, locationName: $locationName, about: $about) {
+  mutation($id: ID!, $slug: String, $name: String, $locationName: String, $about: String) {
+    UpdateUser(id: $id, slug: $slug, name: $name, locationName: $locationName, about: $about) {
```


Put this in file `graphql/Users.js`?